docs/content/composer/user-guide/spaces.md

```diff
@@ -8,10 +8,26 @@ headerDepth: 0
 ::: info Spaces are collections of content that can be shared with others.
 :::
 
-### Personal Space
+## Details
+
+Each object is always in exactly one space.
+
+Spaces don't nest within each other, they're always top-level.
+
+## Types
+
+### Personal space
 
 By default, each user has a personal space that contains all of their content. This space is private to the user and is not shared with anyone else. This space replicates to all of the user's devices.
 
-### Shared Spaces
+### Shared spaces
 
 Users can create shared spaces and invite others to join them. The data in a shared space is replicated across all of the devices of the users who have access to the space.
+
+## Drag-and-drop
+
+::: warning Technology Preview
+Moving objects between spaces is a work in progress.
```

Comment:
This is a somewhat strong statement, but I figured due to https://github.com/dxos/dxos/issues/6749 it was better to set expectations low. We can remove this statement whenever that issue's fixed.

(As far as I can tell drag-and-drop of folders isn't implemented yet either, which is another thing users are going to want immediately. If that's wrong let me know.)